storage/statedb/cache_redis.go

```diff
@@ -0,0 +1,80 @@
+package statedb
+
+import (
+	"errors"
+	"time"
+
+	"github.com/go-redis/redis/v7"
+	"github.com/klaytn/klaytn/common/hexutil"
+)
+
+var (
+	redisCacheDialTimeout = time.Duration(5 * time.Second)
+	redisCacheTimeout     = time.Duration(3 * time.Second)
```

Comment:
In my opinion, this time is too long for redis cache spec.
Please consider to reduce this later.

Reply:
Actually, those values are default values of clustered-disabled redis client. 
um.. How about 2 seconds for Dial and 1 second for Read/Write?

Reply:
500ms and 100ms ?

Reply:
Could you add tests to check if timeout configuration works?

Reply:
@ehnuje Uhm.. Do you want to check the timeout value of redis client can be set with this code or redis sdk works well with timeout options?

Reply:
@aidan-kwon The latter one, to check if the given configuration works with redis sdk.

Reply:
@ehnuje I added a test case, but I think we don't need to have a unit test checking the functionality of redis SDK. So, I disabled after checking the test result.

Reply:
@KimKyungup I changed the timeout and retry conditions. If you have other opinions, please let me know.